### ISSUE TITLE
[Feature] Add "between x and x" to date filter

### ISSUE THREAD

**Issue by migillett** [actualbudget/actual#963]
### Verified feature request does not already exist?

- [X] I have searched and found no existing issue

### 💻

- [X] Would you like to implement this feature?

### Pitch: what problem are you trying to solve?

Current options for filtering dates in the transactions are: `is, is approx, is greater than, is greater than or equals, is less than, and is less than or equals`. Having the ability to filter transactions by a date window would be amazing.

### Describe your ideal solution to this problem

basically just add a button that says "between". When selected, it would pop up another date field. The user adds their specific dates and clicks `apply`. We would need to verify that the dates are not invalid (ie: start date after end date) before making the request.

**Comment by migillett** [comment]
disregard. realized filters are additive. you can filter "between" by adding multiple date filters

**Comment by psybers** [comment]
While that is correct, you can indeed add two separate date filters to get the same effect, I think having an option of 'between' and having it show two date fields at once would be useful.  Even if the end-result is that it just adds two separate filters for you.
